README.md

```diff
@@ -1,19 +1,49 @@
-Absolute
-=============
-
+# Absolute - Web based O2O service platform
 [![Linux/Max Build Status](https://travis-ci.org/lunchclass/absolute.svg?branch=master)](https://travis-ci.org/lunchclass/absolute/branches)
 [![Windows Build status](https://ci.appveyor.com/api/projects/status/099u4iekeny4lpsa/branch/master?svg=true)](https://ci.appveyor.com/project/romandev/absolute/branch/master)
 [![Coverage Status](https://coveralls.io/repos/github/romandev/absolute/badge.svg?branch=master)](https://coveralls.io/github/romandev/absolute?branch=master)
 
-### Welcome to Absolute!
+## Welcome to Absolute!
 
 Absolute is a web-based order platform like siren-order for Starbucks.
 We are focused on creating the platform that is portable, cheaper, easier,
 decentralized and appless.
 
-## Documentation
+**Easier:**  No 'App' install but you can use service. You don't need create account and login any more
+
+**Cheaper:** Absloute is cheaper than other platforms. Because you can just use it if you want.
+
+**Portable:** Absloute support multiple os platform. So, The services you have developed can be used by multiple operating systems.
+
+### Architecture
+<img src="./contribution/image/architecture.png" alt="Absolute Flow" height="200" >
+
+
+### Major Features
+
+* [Web Client -Service-worker](./client/service-worker.md)
+* [Server - Push & oder](./server/server.md) 
+* [Server - Payment](./payment.md)
+
+### Control Flow
+<img src="./contribution/image/flow.png" alt="Absolute Flow" height="200" >
+
+##### Without 'Application'
+
+
+
+## Contributing to Absolute
+Contributions to Absolute are welcomed and encouraged! Please see the
+[Contributing to Absolute guide](./contribution/contribution.md)
 
 ## Getting Started
 
+### Building Absolute
+./absolut start
+
+## Testing Absolute
```

Comment:
Should be ### instead of ##.  if the "Testing Absolute" section is a sub section in "Getting Started".